config/thisroot.sh

```diff
@@ -91,10 +91,12 @@ fi
 
 if [ -z "${MANPATH}" ]; then
    # Grab the default man path before setting the path to avoid duplicates
-   if `which manpath > /dev/null 2>&1` ; then
+   if [[ -n "$(command -v 'manpath')" ]]; then
```

Comment:
```bash
if command -v manpath >/dev/null; then
    default_manpath=$(manpath 2>/dev/null)
fi
```
Another alternative is to check if `${ROOTSYS}/man` is already in `${MANPATH}` or not:
```bash
if ! grep -q "\(^\|:\)${ROOTSYS}/man\(:\|$\)" <<< "$MANPATH" >/dev/null; then
    MANPATH="${MANPATH}:${ROOTSYS}/man"
fi
```

Reply:
thanks for looking at this! i'm not sure what you want me to do..? :smile: 

```bash
if command -v manpath >/dev/null; then
```
feels slightly more hacky than
```bash
if [[ -n "$(command -v 'manpath')" ]]; then
```
to me (although slightly heavier to read), but I'm certainly not a bash expert. If you prefer that formulation I can change it.

Reply:
Why do you think it's more hacky? It's less to type and read :-) I'm just making a suggestion.